src/components/20-molecules/file-upload/ui.test.js

```diff
@@ -250,3 +250,39 @@ test('should exceed maximum size of all files', async t => {
   );
   await t.expect($errorWrapper.innerText).eql('File sizes exceed maximum size');
 });
+
+fixture('File upload - maxSizeOfSingleFileKB prop').page(
```

Comment:
please adjust `'File upload - maxSizeOfSingleFileKB prop'`to something specific. (could be a copy paste mistake)